iroh-net/src/lib.rs

```diff
@@ -1,16 +1,16 @@
+//! iroh-net provides connectivity for iroh.
+//!
+//! This crate is a collection of tools to establish direct connectivity between peers.  At
```

Comment:
isn't the `direct` technically false? as a lot of traffic is relayed?

Reply:
good call